monitoring/godas_plotobs.py

```diff
@@ -0,0 +1,213 @@
+#!/home/gvernier/anaconda3/bin/python
```

Comment:
Does not belong in this PR ... I just merge #272 , update the develop branch in your sand box and merge it in to your branch.

Reply:
resolved